### DISCUSSION TITLE
Transferin and transferout concept in claims

### DISCUSSION THREAD

**Discussion by Sharita23** [synthetichealth/synthea#1292] [Q&A]
I am coming across this transferin and transferout variables in claims data for the first time. How is this related to the actual claims data processing. If you can provide some realtime scenarios that will be great.

Thanks in advance!

Regards,
Sharita

**Comment by jawalonoski** [comment]
Documentation the CSV table is here: https://github.com/synthetichealth/synthea/wiki/CSV-File-Data-Dictionary#claims-transactions

Each claim transaction record has a `type`:
- CHARGE: original line item. 
- PAYMENT: payment made against a charge by an insurance company (aka Payer) or patient.
- ADJUSTMENT: change in the charge without a payment, made by an insurance company. 
- TRANSFERIN and TRANSFEROUT: transfer of the balance from one insurance company to another, or to a patient.

Typically, a single claim will have multiple transactions, some of which are optional...

For example, one set of transactions might track the following claim processing steps:

1. Charge
2. Copay
3. Adjustment
4. Payment by Insurance
5. Transfer to Secondary Insurance
6. Payment by Secondary Insurance
7. Transfer to Patient
8. Payment by Patient
